### DISCUSSION TITLE
Welcome to Pyramid Discussions!

### DISCUSSION THREAD

**Discussion by mmerickel** [Pylons/pyramid#3743] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by zupo** [comment]
This is amazing, thanks Michael!

I've been in the Pyramid space for well over a decade, came in via Plone.

In my company niteo.co, we build every backend with Pyramid, it's so nice to be able to start super small and then grow large on a single framework.

I am also the author of pyramid_openapi3 package and I like to organize events. If there is any interest in a Pyramid sprint/hackathon, lets make it happen!

**Comment by seanh** [comment]
Hi 👋 

I'm a lead developer at [Hypothesis](https://web.hypothes.is/) where I've been for almost a decade. We have several Pyramid apps (https://github.com/hypothesis/h, https://github.com/hypothesis/via, https://github.com/hypothesis/checkmate, https://github.com/hypothesis/bouncer, https://github.com/hypothesis/lms) and extensions (https://github.com/hypothesis/pyramid-sanity, https://github.com/hypothesis/h-pyramid-sentry, https://github.com/hypothesis/pyramid-googleauth, https://github.com/hypothesis/h-assets) as well as a cookiecutter (https://github.com/hypothesis/cookiecutters) that we use to maintain the plumbing in those apps and to generate new apps.

Before Hypothesis I was a CKAN core developer (https://ckan.org/). I think CKAN uses Flask nowadays but back when I worked there it was Pylons.

**Comment by kskarthik** [comment]
Is pyramids actively maintained ?

**Reply by zupo** [reply]
Sure! Why do you think it is not? Any major bugs / security issues are promptly fixed and released: https://pypi.org/project/pyramid/#history

**Comment by russellballestrini** [comment]
I build open source apps using Pyramid.

* https://www.remarkbox.com
    * https://git.unturf.com/engineering/remarkbox/remarkbox

* https://www.makepostsell.com    
    * https://git.unturf.com/engineering/make-post-sell/make_post_sell
    
 I mostly consider myself a consumer of Pyramid but it's my favorite web application framework. I started with the community using Pylons and got help from people on IRC in freenode a lot.
